README.md

```diff
@@ -2,9 +2,9 @@
 
 A Go library that implements a Sparse Merkle tree for a key-value map. The tree implements the same optimisations specified in the [Libra whitepaper](https://developers.libra.org/docs/assets/papers/the-libra-blockchain/2020-05-26.pdf), to reduce the number of hash operations required per tree operation to O(k) where k is the number of non-empty elements in the tree.
 
-[![Tests](https://github.com/lazyledger/smt/actions/workflows/test.yml/badge.svg)](https://github.com/lazyledger/smt/actions/workflows/test.yml)
-[![Coverage Status](https://coveralls.io/repos/github/lazyledger/smt/badge.svg?branch=master)](https://coveralls.io/github/lazyledger/smt?branch=master)
-[![GoDoc](https://godoc.org/github.com/lazyledger/smt?status.svg)](https://godoc.org/github.com/lazyledger/smt)
+[![Tests](https://github.com/celestiaorg/smt/actions/workflows/test.yml/badge.svg)](https://github.com/celestiaorg/smt/actions/workflows/test.yml)
+[![Coverage Status](https://coveralls.io/repos/github/celestiaorg/smt/badge.svg?branch=master)](https://coveralls.io/github/celestiaorg/smt?branch=master)
```

Comment:
Note: this will likely break the integration on rename.

Reply:
Integration with `lazyledger-core`? Note that these are just links, so even if they're broken it's not a big deal. The actual package wasn't isn't changed in this PR: https://github.com/celestiaorg/smt/blob/738728c76032ff7650301b005dda2beaa6c5ad80/go.mod#L1

Reply:
No, integration with coveralls.

Reply:
Oh I see. Okay I can revert this one line for now.

Reply:
Reverted in 49a51bb

Reply:
Filed #41 to handle rebrand.

Reply:
No, this was just a note, rather than a request to change it back. It should be changed to celestiaorg.